src/Stepper/Stepper.less

```diff
@@ -1,18 +1,10 @@
 @import "../less/fonts.less";
 @import (reference) "../less/index";
 
-.Stepper--stickyContainer {
-  .flexbox();
-}
-
-.Stepper--stickyContent {
-  .flex(0, 1, 18rem);
-  max-width: 18rem;
-}
-
 .Stepper--stepsDisplay {
   .list-reset;
   .padding--l;
+  max-width: 18rem;
```

Comment:
i think we should leave this out for now - this is something that can be determined by the client that's rendering the stepper